names.h

```diff
@@ -9,6 +9,7 @@
 #define _NAMES_H
 
 #include <sys/types.h>
+#include <stdint.h>
```

Comment:
Well it's needed in the header, that's why `<sys/types.h>` is also there, because those types are used there.
Won't compile if it's in `names.c`

How should we deal with `u_int8_t` and `uint8_t`? Is it okay to use both right now? The libusb function I call returns `uint8_t` - that's why I'm using it.

Reply:
use what libusb is using, we should unify everything eventually, don't worry about ti for now.  so this is fine, sorry for the noise.